MedscribeUI/src/components/NoteControls/NotecontrolsLayout.tsx

```diff
@@ -0,0 +1,78 @@
+import { Select, Button } from '@mantine/core';
+import { useState } from 'react';
+import ThreeBtnSelector from '../Utilities/ThreeBtnSelector';
+
+const visitSelectorLabel = 'Visit';
+const defaultPronoun = 'He';
+
+function NoteControlsLayout() {
+  const [selectedPronoun, setSelectedPronoun] = useState(defaultPronoun);
+  const [selectedNoteLength, setSelectedNoteLength] = useState('Standard');
+  const [selectedVisitType, setSelectedVisitType] = useState('New Patient');
+
+  const handleVisitTypeSelect = (value: string | null) => {
+    setSelectedVisitType(value || ''); // Set default value to empty string if value is null
+    console.log('Selected Visit Type:', value); // !!!DEBUG ONLY!!! - Remove before production
+  };
+
+  const handlePronounSelect = (value: string) => {
+    setSelectedPronoun(value);
+    console.log('Selected Pronoun:', value); // !!!DEBUG ONLY!!! - Remove before production
+  };
+
+  const handleNoteLengthSelect = (value: string) => {
+    setSelectedNoteLength(value);
+    console.log('Selected Note Length:', value); // !!!DEBUG ONLY!!! - Remove before production
+  };
+
+  const handleMagicEdit = () => {
+    console.log('Magic Edit Clicked'); // !!!DEBUG ONLY!!! - Remove before production
```

Comment:
We are replacing the button that this is attached to with the regenerate new report button so just change this up accordingly a bit

Reply:
As in remove the function in total, or just leave an empty function? I just made it an empty function so we can hook it up later and removed all debugging things